microbench/microbench_all.cc

```diff
@@ -0,0 +1,17 @@
+//  Copyright (c) 2011-present, Facebook, Inc.  All rights reserved.
+//  This source code is licensed under both the GPLv2 (found in the
+//  COPYING file in the root directory) and Apache 2.0 License
+//  (found in the LICENSE.Apache file in the root directory).
+
+// this is a simple micro-benchmark for compare ribbon filter vs. other filter
+// for more comprehensive, please check the dedicate util/filter_bench.
+
+#include <benchmark/benchmark.h>
+
+#include "microbench/db_basic_bench.cc"
+#include "microbench/ribbon_bench.cc"
```

Comment:
I think it could be a problem as we adding more benchmark files, like name conflict, especially because each benchmark code includes its own header files.
Questions:
- [ ] do we have to have single benchmark binary?
- [ ] can we replace it with a script which calls other binaries?

Reply:
> do we have to have single benchmark binary?

I don't think so.
> can we replace it with a script which calls other binaries?

I don't think so.

I'm fairly confident I can get this to work well while having each benchmark file be a separate binary (provided each benchmark has a binary_cpp target), so I'll try this.